core/tauri-build/src/mobile.rs

```diff
@@ -5,17 +5,22 @@
 use std::{fs::write, path::PathBuf};
 
 use anyhow::{Context, Result};
+use semver::Version;
+use tauri_utils::config::Config;
 
-pub fn generate_gradle_files(project_dir: PathBuf) -> Result<()> {
+pub fn generate_gradle_files(project_dir: PathBuf, config: &Config) -> Result<()> {
   let gradle_settings_path = project_dir.join("tauri.settings.gradle");
   let app_build_gradle_path = project_dir.join("app").join("tauri.build.gradle.kts");
+  let app_gradle_properties_path = project_dir.join("app").join("gradle.properties");
```

Comment:
let's pick a different name, `gradle.properties` already exists in the android project even if it is in another directory
```suggestion
  let app_gradle_properties_path = project_dir.join("app").join("tauri.properties");
```